spec/features/user_signs_up_spec.rb

```diff
@@ -40,7 +41,7 @@
     click_button "Sign up"
 
     # within[id]
-    # expect(find('img')['src']).to have_content "default_av.jpg"
+    expect(find('img')['src']).to have_content "default_av.jpg"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.